code/game/objects/items/singularityhammer.dm

```diff
@@ -53,7 +53,13 @@
 			var/atom/movable/A = X
 			if(A == wielder)
 				continue
-			if(A && !A.anchored && !ishuman(X))
+			if(isliving(A))
+				var/mob/living/vortexed_mob = A
+				if(vortexed_mob.mob_negates_gravity())
+					continue
+				else
+					vortexed_mob.Paralyze(2 SECONDS)
```

Comment:
weird seemingly unrelated singularity hammer buff? am i missing something here?

Reply:
i mean it's a buff that... doesn't really affect players but odd change being here